source/api/discovery/0.3/index.md

```diff
@@ -0,0 +1,784 @@
+
+##### id
+
+An identifier for the Activity.
+
+Activities _MAY_ have an `id` property. The value _MUST_ be a string and it _MUST_ be an HTTP(S) URI. The JSON representation of the Activity _MAY_ be available at the URI.
+
+```
+{ "id": "https://example.org/activity/1" }
+```
+
+##### type
+
+The type of Activity. 
+
+This specification uses the types described in the table below.
+
+| Type   | Definition |
+| ------ | ---------- |
+| `Create` | The initial creation of the resource.  Each resource _MUST_ have at most one `Create` Activity in which it is the `object`. |
+| `Update` | Any change to the resource.  In a system that does not distinguish creation from modification, then all changes _MAY_ have the `Update` type. |
+| `Delete` | The deletion of the resource, or its de-publication from the web. Each resource _MUST_ have at most one Activity in which it is the `object`. |
+{: .api-table #table-type-dfn}
+
+Activities _MUST_ have the `type` property. The value _MUST_ be a registered Activity class, and _SHOULD_ be one of `Create`, `Update`, or `Delete`.
+
+```
+{ "type": "Update" }
+```
+
+##### object
+
+The IIIF resource that was affected by the Activity.  It is an implementation decision whether there are separate lists of Activities, one per object type, or a single list with all of the object types combined.
+
+Activities _MUST_ have the `object` property.  The value _MUST_ be a JSON object, with the `id` and `type` properties.  The `id` _MUST_ be an HTTP(S) URI. The `type` _SHOULD_ be a class defined in the IIIF Presentation API, and _SHOULD_ be one of `Collection`, or `Manifest`. The object _MAY_ have a `seeAlso` property, as defined for `OrderedCollection` above, to reference a description document of the object resource. The document referenced in the `seeAlso` property _MAY_ also be referenced with the `seeAlso` property in an instance of the IIIF Presentation API.
+
+<!-- This can't be on the Activity, as some activities reference more than one resource e.g. Add X to Y, Move X from Y to Z, etc. -->
+
+
+```
+{
+  "object": {
+    "id": "http://example.org/iiif/1/manifest",
+    "type": "Manifest",
+    "seeAlso": [
+      {
+        "id": "https://example.org/dataset/single-item.jsonld",
+        "type": "Dataset",
+        "format": "application/ld+json"
+      }
+    ] 
+  }
+}
+```
+
+##### endTime
+
+The time at which the Activity was finished. It is up to the implementer to decide whether the Activity includes the publication of the IIIF resource online, or only the internal data modification, but the decision _MUST_ be consistently applied. The changed resource given in `object` _MUST_ be available at its URI at or before the timestamp given in `endTime`. The value of `endTime` _SHOULD_ be before the time that the Activity is published as part of its Ordered Collection.
```

Comment:
How can an implementer only do "the internal data modification" (not publish it) _AND_ have "changed resource given in `object` _MUST_ be available at its URI at or before the timestamp given in `endTime`" ? I would strike the second sentence.

Reply:
Sentence is opaque, I agree.
Clarified, I hope, to: 
> ... whether the value of `endTime` is the timestamp for the publication of the IIIF resource online or is the timestamp of the modification to the data in the managing system if these are different,  ...

Reply:
:+1: